src/main/java/htsjdk/samtools/util/IOUtil.java

```diff
@@ -87,7 +87,10 @@
     public static final long FIVE_GBS = 5 * ONE_GB;
 
     /** Possible extensions for VCF files and related formats. */
```

Comment:
This only applies to the field `VCF_EXTENSION`. Maybe it should be moved down to the line before `VCF_EXTENSIONS`. And you could add javadocs for the other new public fields.